src/uproot/compression.py

```diff
@@ -7,8 +7,8 @@
 
 from __future__ import annotations
 
+import math
 import struct
-import threading
```

Comment:
The Python ZSTD library is thread-unsafe, so it's good to be able to remove this now.